### DISCUSSION TITLE
I have problem with importing data from json file to my python code!

### DISCUSSION THREAD

**Discussion by AminAiden** [Rapptz/discord.py#9424] [Q&A]
I was trying make a discord bot and it needs to import token and application id from a json file but it returns this error:`Traceback (most recent call last):
  File "C:\Users\parsin\PycharmProjects\RubbyDigger\rubbybot\rubbydiggerbot.py", line 34, in <module>
    bot = Bot(config)
  File "C:\Users\parsin\PycharmProjects\RubbyDigger\rubbybot\rubbydiggerbot.py", line 11, in __init__
    application_id=config("application-id")
TypeError: 'dict' object is not callable`

if you know how can I fix it please don't be shy🙏

**Comment by No767** [comment]
From what I can tell, you probably aren't doing it correctly. Copy and paste your code into [mystbin](https://mystb.in/) and link it here. Without your code, I can't tell what's wrong.

**Comment by CaedenPH** [comment]
> I was trying make a discord bot and it needs to import token and application id from a json file but it returns this error:`Traceback (most recent call last):
> 
>   File "C:\Users\parsin\PycharmProjects\RubbyDigger\rubbybot\rubbydiggerbot.py", line 34, in <module>
> 
>     bot = Bot(config)
> 
>   File "C:\Users\parsin\PycharmProjects\RubbyDigger\rubbybot\rubbydiggerbot.py", line 11, in __init__
> 
>     application_id=config("application-id")
> 
> TypeError: 'dict' object is not callable`
> 
> 
> 
> if you know how can I fix it please don't be shy🙏

Try `Bot(**config)` to unpack your config dict
